operator_ui/src/components/Dashboards/Activity.tsx

```diff
@@ -130,6 +146,17 @@ const RecentActivity = ({ classes, runs }) => {
             </TableRow>
           ))}
         </TableBody>
+        {count > pageSize && (
+          <TableFooter>
```

Comment:
I think we're using this for 2-3 tables already, so would it benefit from an extraction?

Reply:
on the second thought, it might be better to postpone that for when code sharing & styleguide is a thing and we make Table a generic extracted component.